t/34-developer_mode-unit.t

```diff
@@ -94,17 +94,14 @@ sub prepare_waiting_for_finished_handled {
 }
 sub wait_for_finished_handled {
     # wait until the finished event is handled (but at most 5 seconds)
-    if (!$finished_handled) {
-        my $timer = Mojo::IOLoop->timer(
-            5.0 => sub {
-
-            });
-        Mojo::IOLoop->one_tick;
-        Mojo::IOLoop->remove($timer);
-    }
-
+    my $timer = Mojo::IOLoop->timer(
+        5.0 => sub {
+            Mojo::IOLoop->stop;
```

Comment:
I tested this as well as the state of master and tests do not reliably cover this line but you can just add `# uncoverable statement` at the end of line, e.g.:

```perl
Mojo::IOLoop->stop; # uncoverable statement
```

Reply:
Yes, ironically it made no difference to an empty block. However `sub{}` seems to work even better